src/QueryManager.ts

```
@@ -701,6 +707,9 @@ export class QueryManager {
 
   private startQuery(queryId: string, options: WatchQueryOptions, listener: QueryListener) {
     this.queryListeners[queryId] = listener;
+    if (options.noFetch) {
+      options.pollInterval = null;
```

If someone passes `noFetch` and a `pollInterval`, I assume we should throw an error rather than silently ignoring it.